tests/rpmquery.at

```diff
@@ -180,6 +180,34 @@ runroot rpmspec \
 [])
 RPMTEST_CLEANUP
 
+AT_SETUP([rpmspec -q --rpms])
+AT_KEYWORDS([query])
+RPMTEST_CHECK([
+RPMDB_INIT
+runroot rpmspec --rpms \
+  -q  /data/SPECS/hello.spec | grep src
+runroot rpmspec --rpms \
+  -q  /data/SPECS/hello.spec | cut -d. -f-2
```

Comment:
Put these two checks into separate RPMTEST_CHECK() blocks, it makes the tests far more reliable when you don't need to guess which part output what. There are a bunch of tests that do this before we learned that you can have many checks inside a setup/clean pair :sweat_smile:  but it's a bad practise we should not do in new tests. 
Also, these are not writing anything so you don't need RPMDB_INIT (which has non-zero cost).